code/modules/wiremod/integrated_circuit.dm

```diff
@@ -476,6 +506,51 @@ GLOBAL_LIST_EMPTY_TYPED(integrated_circuits, /obj/item/integrated_circuit)
 			. = TRUE
 		if("save_circuit")
 			return attempt_save_to(usr.client)
+		if("add_variable")
+			var/variable_identifier = copytext(params["variable_name"], 1, PORT_MAX_NAME_LENGTH)
```

Comment:
```suggestion
			var/variable_identifier = trim(copytext(params["variable_name"], 1, PORT_MAX_NAME_LENGTH))
```

Otherwise, you'll be able to make an empty name with a bunch of spaces.